packages/application-components/src/components/dialogs/info-dialog/info-dialog.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CloseIcon,
+  Constraints,
+  SecondaryIconButton,
+  Spacings,
+  Text,
+} from '@commercetools-frontend/ui-kit';
+import ModalContainer from '../modal-container';
+import styles from './info-dialog.mod.css';
+
+// Like "Spacings.Inline", but with the `justify-content: space-between` style
+const Stretched = props => (
+  <div className={styles.stretched}>{props.children}</div>
+);
+Stretched.displayName = 'Stretched';
+Stretched.propTypes = { children: PropTypes.node.isRequired };
+
+const InfoDialog = props => (
+  <ModalContainer
+    isOpen={props.isOpen}
+    closeTimeoutMS={props.closeTimeoutMS}
+    onRequestClose={props.onClose}
```

Comment:
we have onClose here but also in the secondary icon button. Does that mean it will be called twice? Once immediately, and once after 150ms?

Reply:
No, the dialog can be closed in 3 ways, all of them map to the `onClose function`
- clicking on the overlay (modal triggers `onRequestClose`)
- pressing ESC (modal triggers `onRequestClose`)
- clicking on the close button (secondary button triggers `onClick`)

Reply:
In this case should we remove the leave delay functionality since it doesn't apply when you click the secondary button? Or add a setTimeout so it matches the other cases?

Reply:
Hmm good point, I haven't thought about it. I guess we can remove it then, I don't see much the value on having that.